docs/advanced-guide/filtered-logging/page.md

```diff
@@ -0,0 +1,72 @@
+# Filtered Logging
+
+Filtered logging is a feature that allows you to mask sensitive data in your log messages. It helps protect personally identifiable information (PII) and other confidential data from being exposed in logs.
+
+## Usage
+
+To use filtered logging, you need to set the masking configuration in your `.env` file. The following configuration options are available:
+
+- `LOGGER_MASKING_ENABLED`: Set to `true` to enable masking of sensitive fields in log messages. Default is `false`.
+- `LOGGER_MASKING_FIELDS`: A comma-separated list of field names that should be masked in log messages.
```

Comment:
@SuyeshBadge - This use-case can be fulfilled with a single config, not required to create a bool config as well. Please fix this.